src/appleseed/renderer/kernel/lighting/pt/ptlightingengine.cpp

```diff
@@ -1043,8 +1043,8 @@ Dictionary PTLightingEngineFactory::get_params_metadata()
         "max_diffuse_bounces",
         Dictionary()
             .insert("type", "int")
-            .insert("default", "8")
-            .insert("unlimited", "true")
+            .insert("default", "3")
+            .insert("unlimited", "false")
```

Comment:
That should not change as well.

That said, I've just realized that the `unlimited` property is completely unused to I'll remove it altogether.

Reply:
To clarify: what it's supposed to mean that the property's value _can_ be unlimited. The name is less than ideal, I agree.